src/bosh-google-cpi/util/util.go

```diff
@@ -30,8 +29,10 @@ func RegionFromZone(zone string) string {
 	return ""
 }
 
+var zoneRe = regexp.MustCompile("/zones/([a-zA-Z1-9-]+)/?")
```

Comment:
The linter was complaining about the unnecessary types in the declaration statement, so I cleaned them up. Also moved  each closer to its function of concern for clarity.